### PR TITLE
Implement single method interfaces

### PR DESCRIPTION
**Pull request by shadowhand**
Refs http-interop/http-factory#1

### CODE REVIEW

File: src/StreamFactory.php [hunk 1]
```diff
@@ -2,16 +2,25 @@
...
 use Zend\Diactoros\CallbackStream;
 
-class StreamFactory implements StreamFactoryInterface
+class StreamFactory implements
+    CallbackStreamFactoryInterface,
+    ResourceStreamFactoryInterface,
+    StreamFactoryInterface
 {
-    public function createStream()
+    public function createStream($body = '')
```
Comment (harikt) [inline src/StreamFactory.php]: Why not make use of the diactoros stream itself ? I am looking at https://github.com/zendframework/zend-diactoros/blob/b4e7758556c97b5bb9a5260d898e9788ee800538/src/Stream.php#L32

Comment (shadowhand) [inline src/StreamFactory.php]: Oops bad copy/paste. That makes perfect sense.

**Comment by Nyholm** [comment]
👍

**Comment by Nyholm** [comment]
Why was this issue closed?

**Comment by shadowhand** [comment]
Oh, I thought it was covered by other changes. I'll reopen.

**Comment by shadowhand** [comment]
@Nyholm refreshed and merged.

**Comment by Nyholm** [comment]
Good! Thank you

**Pull request merged**